src/apply_conf.rs

```diff
@@ -183,29 +183,36 @@ fn copy_connection_files(
 
     for interface in &host.interfaces {
         info!("Processing interface '{}'...", &interface.logical_name);
+        let connections = &interface
+            .connection_ids
+            .clone()
```

Comment:
We can drop this by removing the reference when iterating.

Reply:
I don't understand what you mean.
Can you make a "suggestion"

Reply:
Yes, the following should work:
```
    for interface in host.interfaces {
        info!("Processing interface '{}'...", &interface.logical_name);
        let connections = &interface
            .connection_ids
            .unwrap_or(vec![interface.logical_name.clone()]);
```

Note that the reference (&) from `host.interfaces` is removed.